frontend/src/pages/Login/Login.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { TextField, Button, Typography, makeStyles } from '@material-ui/core';
+import { useHistory, Link } from 'react-router-dom';
+import { setJWT, setUser } from '../../auth';
+import './Login.css'
+const config = require('../../config');
+
+const useStyles = makeStyles((theme) => ({
+  centered: {
+    textAlign: 'center'
+  },
+  form: {
+    //Input Field - General Layout
+    '& .MuiTextField-root': {
+      margin: theme.spacing(1),
+      width: '95%'
+    },
+    "& .MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-notchedOutline": {
+      borderColor: "black"
+    },
+    "& .MuiInputLabel-outlined.Mui-focused": {
+      color: "black"
+    },
+    '& .MuiTypography-root': {
+      margin: theme.spacing(1),
+      width: '100%'
+    },
+
+    '& .MuiButton-root': {
+      margin: theme.spacing(3),
+      background: '#54682f',
+      width: '30%'
+    }
+  },
+  title: {
+    margin: theme.spacing(2),
+    textAlign: 'center',
+    fontWeight: 'bolder',
+    textTransform: 'uppercase'
+  }
+}));
+
+const Login = (props) => {
```

Comment:
Class vs functional components?

Reply:
We used functional components because we want to redirect the user when the login button is clicked. History doesn't support class components.